CRM/Contribute/Form/ContributionRecur.php

```diff
@@ -87,7 +87,7 @@ class CRM_Contribute_Form_ContributionRecur extends CRM_Core_Form {
   /**
    * Details of the subscription (recurring contribution) to be altered.
    *
-   * @var array
+   * @var \CRM_Core_DAO
    */
   protected $subscriptionDetails = [];
```

Comment:
It does seem to be a DAO, so now this initialization is weird.

Otherwise agree with everything in the PR and did a quick r-run seems ok.

Reply:
@demeritcowboy ouch - but TBH I'd rather punt on that as the change is right & I'm not touching this form other than comments

Reply:
Ok